File: programmers/lv0/코딩기초트레이닝/출력/대소문자바꿔서출력하기/Solution.java
package lv0.코딩기초트레이닝.출력.대소문자바꿔서출력하기;

import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String a = sc.next();
        for (char c : a.toCharArray()) {
            if (Character.isUpperCase(c)) {
                System.out.print(Character.toLowerCase(c));
            }
            else {
                System.out.print(Character.toUpperCase(c));
            }
        }
    }
}

File: programmers/lv0/코딩기초트레이닝/출력/대소문자바꿔서출력하기/Solution.kt
package lv0.코딩기초트레이닝.출력.대소문자바꿔서출력하기

fun main(args: Array<String>) {
    val s1 = readln()
    val convertedString = s1.toCharArray().map { c ->
        if (c.isUpperCase()) {
            c.lowercaseChar()
        } else {
            c.uppercaseChar()
        }
    }.joinToString("")

    println(convertedString)
}
